members/SMNU33807_524016.yaml

```diff
@@ -0,0 +1,57 @@
+#MN State Senator Andrew Mathews
+bioguide: SMNU33807_524016
+contact_form:
+  driver: chrome
+  steps:
+    - visit: https://www.senate.mn/members/email-form/1259
```

Comment:
Looks like this is the URL for Liz Bolden.  I believe Mathews is https://www.senate.mn/members/email-form/1222